test_harness.py

```diff
@@ -61,6 +62,13 @@ def _run_command(num_devices, command):
     :param list command: the command to be run
     """
     devices = _make_loopbacked_devices(num_devices)
+    for device in devices:
+        for option in ["--getss", "--getpbsz", "--getiomin", "--getioopt"]:
+            diagcmd = ["blockdev", option, device]
+            with subprocess.Popen(diagcmd, stdout=subprocess.PIPE) as proc:
+                blockdevoutput = proc.stdout.readline().strip().decode("utf-8")
+            logging.debug("output of %s: %s", diagcmd, blockdevoutput)
```

Comment:
Try to use f-strings here.

Reply:
I tried before, but it failed linting with this:
```
test_harness.py:70:12: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)
```

Reply:
Ok!